src/shared/directives/focus-element/focus-element.directive.ts

```diff
@@ -0,0 +1,20 @@
+import { AfterViewInit, Directive, ElementRef, Renderer2 } from '@angular/core';
+
+@Directive({
+  selector: '[focusElement]'
+})
```

Comment:
can we add a little description to explain what this directive does?

Reply:
completely missed that part 😁. Will add now

Reply:
done now